src/hu/elte/txtuml/stdlib/util/DictionaryInterface.java

```diff
@@ -0,0 +1,137 @@
+package hu.elte.txtuml.stdlib.util;
+
+	
+import java.util.Enumeration;
+
+	import hu.elte.txtuml.api.model.external.ExternalClass;
+
+
+	/**
+	 * This class provides the functionality of the dictionary Java class for txtUML models.
+	 * <p>   
+     * The <code>Dictionary</code> class is the abstract parent of any
+     * class, such as <code>Hashtable</code>, which maps keys to values.
+     * Every key and every value is an object. In any one <tt>Dictionary</tt>
+     * object, every key is associated with at most one value. Given a
+     * <tt>Dictionary</tt> and a key, the associated element can be looked up.
+     * Any non-<code>null</code> object can be used as a key and as a value.
+     * <p>
+     * As a rule, the <code>equals</code> method should be used by
+     * implementations of this class to decide if two keys are the same.
+     * <p>
+     * <strong>NOTE: This class is obsolete.  New implementations should
+     * implement the Map interface, rather than extending this class.</strong>
+     *
+     * @author  Kovan Rashid
+     * @see     java.lang.Object#hashCode()
+     * @see     java.util.Hashtable
+     */
+	
+		public interface DictionaryInterface extends ExternalClass{
```

Comment:
The interface's name should be the name of the represented Java class, `Dictionary`.